include/ipc/ipc_service.h

```diff
@@ -116,6 +116,19 @@ struct ipc_ept_cfg {
 	void *priv;
 };
 
+/** @brief Open an instance
+ *
+ *  Optional function to be used to open an instance before being able to
+ *  register a new endpoint on it.
+ *
+ *  @retval -EIO when no backend is registered.
+ *  @retval -EINVAL when instance or endpoint configuration is invalid.
```

Comment:
I think at this stage we don't have any endpoints to worry about.
```suggestion
 *  @retval -EINVAL when instance configuration is invalid.
```